docs/development/releasing.rst

```diff
@@ -89,6 +84,16 @@ packages that use the full bugfix/maintenance branch approach.)
 
       $ git checkout v1.2.x
 
+#. Make sure that the continuous integration services (e.g., Travis) are passing
+   for the `astropy core repository`_ branch you are going to release. You may
+   also want to locally run the tests (with remote data on to ensure all of the
+   tests actually run), and make sure the description in ``setup.cfg`` is reStructuredText-
+  compliant::
```

Comment:
Ops... I think it was underindented. Will fix in the commit.
```suggestion
   compliant::
```

Reply:
oops I meant to make reStructureText-compliant one hyphenated word, but wasn't sure how to do that here.

Reply:
That is a limitation of this "suggestion" feature on GitHub. It does not do well with multi-line editing. But no matter, I'll sort this out in my next batch of edits. Thank you!